src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -399,9 +399,6 @@ public void testViewPhotoAndAjaxForLargeScaledSession() throws Exception {
         resultsPage.removeNavBar();
         resultsPage.hoverClickAndViewStudentPhotoOnHeading(5, "studentProfilePic?studentemail={*}&courseid={*}&user=CFResultsUiT.instr");
         resultsPage.hoverAndViewStudentPhotoOnBody(5, "studentProfilePic?studentemail={*}&courseid={*}&user=CFResultsUiT.instr");
-        resultsPage.click(By.id("panelHeading-5"));
-        ThreadHelper.waitFor(1000);
-
```

Comment:
These are added back in 1926b89 (too long ago, maybe @thyageshm can't even remember why) and I find no plausible reason for it. We are not testing panel collapse here (and if we did this is a wrong way to do it), and if the intention is to bring something to the viewport, my other argument stands: as long as we don't play with element visibility (which we don't here), nothing needs to be on the viewport.
I haven't really found out why this causes the subsequent `hoverClickAndViewStudentOnPhotoHeading` to fail, but based on the above reasoning I'm confident enough that removing these problematic lines are the way to go.